parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -444,6 +452,8 @@ def set_stream_logger(name='parsl', level=logging.DEBUG, format_string=None):
                         help="Number of cores assigned to each worker process. Default=1.0")
     parser.add_argument("-t", "--task_url", required=True,
                         help="REQUIRED: ZMQ url for receiving tasks")
+    parser.add_argument("--max_workers", default=float(int),
```

Comment:
1. Should be `default=float('inf')`.
2. But even with `float('inf')`, it returns error. On line 490, `int(max_workers)` would return error like `cannot convert float infinity to int`. I guess default to a large enough number is enough though it is not ideal?

Reply:
Either:   use `None` as a default and check that appropriately later on;   or make this a required argument by adding `required = True`, at which point the default value becomes (I think) irrelevant.

Using arbitrary magic large numbers makes me sad.

Reply:
I believe this should be resolved in code now. By default the user specifies resources/cores per worker, and max_workers is available to limit the # of workers launched. It defaults to float('inf') and when specified an int is expected.